src/components/editTestCase/qdm/EditTestCase.tsx

```diff
@@ -80,14 +82,23 @@ const EditTestCase = () => {
       showToast("Error while calculating QDM test cases", "danger");
     }
   };
-
+  const {
+    title = "",
+    description = "",
+    series = "",
+    json = "",
+  } = currentTestCase || {};
   const formik = useFormik({
     initialValues: {
-      ...currentTestCase,
+      title,
+      description,
+      series,
+      json,
+      id,
     },
-    // validationSchema: QDMPatientSchemaValidator, to do
+    validationSchema: QDMPatientSchemaValidator,
     enableReinitialize: true,
-    onSubmit: (currentTestCase: TestCase) => {
+    onSubmit: (currentTestCase: any) => {
```

Comment:
was there any issue with the using `TestCase` type?

Reply:
Yes, the initial values no longer hold every part property on the test case object. just the parts that are being manipulated. this now throws properties missing